src/test/scala/org/broadinstitute/hail/driver/FilterAllelesSuite.scala

```diff
@@ -0,0 +1,117 @@
+package org.broadinstitute.hail.driver
```

Comment:
Ah hmm. [`FilterVariantsSuite`](https://github.com/broadinstitute/hail/blob/master/src/test/scala/org/broadinstitute/hail/driver/FilterVariantsSuite.scala) is in `org.broadinstitute.hail.driver`. If we move `FilterAllelesSuite` I would move `FilterVariantsSuite` as well.

Generally, I'd keep tests in the same package as the things they test.